src/cookbook/navigation/set-up-universal-links.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Set up universal links for iOS
+description: How to animate a widget from one screen to another
+prev:
+title: Send data to a new screen
+path: /cookbook/navigation/passing-data
+next:
+title: Delete data on the internet
+path: /cookbook/networking/delete-data
+js:
+- defer: true
+  url: https://dartpad.dev/inject_embed.dart.js
+---
+
+<?code-excerpt path-base="cookbook/navigation/deeplink"?>
+
+Deeplink is a mechanism to launch an app with a uri. This uri
+contains scheme, host, and path and leads users to a specific
+screen of an app.
+
+A universal link is a deeplink with http or https scheme that
+is exclusive to Apple devices.
+
+
+## Prerequisite
+1. A mac with Flutter SDK and Xcode installed
+2. A web domain
```

Comment:
We don't typically include this for a recipe.

```suggestion
```

We do typically write a short intro and list the broad steps. For example:
```
The “swipe to dismiss” pattern is common in many mobile apps. For example, when writing an email app, you might want to allow a user to swipe away email messages to delete them from a list.

Flutter makes this task easy by providing the [Dismissible](https://api.flutter.dev/flutter/widgets/Dismissible-class.html) widget. Learn how to implement swipe to dismiss with the following steps:

1. Create a list of items.
2. Wrap each item in a Dismissible widget.
3. Provide “leave behind” indicators.
```